.github/workflows/continuous-integration-workflow.yml

```diff
@@ -55,3 +56,8 @@ jobs:
         run: DESTDIR=${PWD}/install cmake --build builddir --target install && rm -rf ${PWD}/install/usr && rmdir ${PWD}/install
       - name: Install
         run: sudo cmake --build builddir --target install
+      - name: Test install
+        working-directory: example/tutorial/01_hello_world
```

Comment:
The Jenkins CI uses `build_cmake_installed` instead:
https://github.com/kokkos/kokkos/blob/af5d246e933065e68ed40d8b1a1e8fccfd7b6ae4/.jenkins#L237-L276

Reply:
Nice let's use that.